### PR TITLE
webp-to-png@tomunek: create webp-to-png@tomunek nemo action

### PR DESCRIPTION
**Pull request by Tomunek**
I hate that every time I download an image from the internet I get a WEBP and have to go to an online converter or use the terminal to turn it into a PNG.

I created an action to solve this issue.

### CODE REVIEW

File: webp-to-png@tomunek/webp-to-png@tomunek.nemo_action.in [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[Nemo Action]
+_Name=Convert to PNG
+_Comment=Convert selected WEBP file to PNG
+Exec=<webp-to-png@tomunek/webp-to-png@tomunek.sh %F>
+Icon-Name=document-properties-symbolic
+Selection=s
```
Comment (rcalixte) [inline webp-to-png@tomunek/webp-to-png@tomunek.nemo_action.in:6]: You could theoretically expand this to cover multiple files at some point if you want.

**Comment by rcalixte** [comment]
Nothing wrong with this action but there are software solutions for this too, one nice one being Converseen:
* [Main site](https://converseen.fasterland.net/)
* [GitHub repository](https://github.com/Faster3ck/Converseen)
* [Flathub page](https://flathub.org/apps/net.fasterland.converseen)

**Review by rcalixte — APPROVED** [review]

File: webp-to-png@tomunek/webp-to-png@tomunek.nemo_action.in (hunk 1 above)
Comment (Tomunek) [reply webp-to-png@tomunek/webp-to-png@tomunek.nemo_action.in:6]: I am considering adding support for WEBM (to MP4).
Is it allowed to change the name of the extension (for example to webconv) if I decide to expand it's functionality?

Comment (rcalixte) [reply webp-to-png@tomunek/webp-to-png@tomunek.nemo_action.in:6]: Of course! The action name itself is the only thing that is stuck but the functionality can be expanded as you would like.

**Pull request merged**